docs/arrays.cpp

```diff
@@ -0,0 +1,36 @@
+#include "arrays.h"
+
+using namespace std;
```

Comment:
```suggestion:+1

```

![image](https://user-images.githubusercontent.com/42613926/106993550-f0314880-6748-11eb-85d1-723a5aa03540.png)

I encountered an issue with `using namespace std;` in `arrays.cpp` which I think comes from not including any standard library functions in the `.cpp` file (according to Google). 

![image](https://user-images.githubusercontent.com/42613926/106993752-6170fb80-6749-11eb-84bf-cb13722ff9e7.png)

Removing the declaration, which I believe is not used, eliminates the warning. There are other solutions (removing `-Werror`, adding a simple `iostream` `#include` or something) but unless I'm missing something this seems the most reasonable.

Reply:
Added a commit to remove that.